bin/lintsite

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env php
+<?php
+
+use Symfony\Component\Console\Input\ArgvInput;
+
+date_default_timezone_set('UTC');
+
+set_time_limit(0);
+
+(@include_once __DIR__ . '/../vendor/autoload.php') || @include_once __DIR__ . '/../../../autoload.php';
+
+use Symfony\Component\Console\Application;
+
+$application = new Application('TwigYard', '0.1.0');
+
+$application->add(new Symfony\Component\Yaml\Command\LintCommand());
+
+$twigLintCommand = new Symfony\Bridge\Twig\Command\LintCommand();
+
+$twigEnvironment = new \Twig_Environment(new \Twig_Loader_Filesystem());
+
+$twigEnvironment->addExtension(new Twig_Extensions_Extension_Text());
+$twigEnvironment->addExtension(new Twig_Extensions_Extension_I18n());
+$twigEnvironment->addExtension(new Twig_Extensions_Extension_Intl());
+$twigEnvironment->addExtension(new Twig_Extensions_Extension_Date());
+$translator = new Symfony\Component\Translation\Translator('en_US');
+$twigEnvironment->addExtension(new Symfony\Bridge\Twig\Extension\TranslationExtension($translator));
+
+$twigEnvironment->addFunction(new Twig_SimpleFunction('asset', function () {}));
+$twigEnvironment->addFunction(new Twig_SimpleFunction('image', function () {}));
+$twigEnvironment->addFunction(new Twig_SimpleFunction('path', function () {}));
+
+$twigLintCommand->setTwigEnvironment($twigEnvironment);
+$application->add($twigLintCommand);
+
+$application->setAutoExit(false);
+
+$argv = $_SERVER['argv'];
+$argc = $_SERVER['argc'];
+
+if ($argc === 2) {
```

Comment:
The paths in this section are too general. For example it checks yaml files in node_modules sit directories.

It needs to check only in src/data and in the site root